### PR TITLE
Update and add billing articles to reflect changes in console

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
Updated content across several articles so that the instructions accurately reflect the new navigation in the web console. The changes include migrating some information from the billing groups article in the help center. Instructions for updating the tax status were also moved from the conceptual article to a new how-to article.

Resolves: #1071

### CODE REVIEW

**Comment by lornajane** [comment]
@staceysalamon-aiven who can review this change for you?

**Comment by staceysalamon-aiven** [comment]
> @staceysalamon-aiven who can review this change for you?

Added PM as reviewer.

**Comment by rsalevsky** [comment]
LGTM but I would like that AP does the final review, he is the expert here.

**Comment by staceysalamon-aiven** [comment]
> LGTM but I would like that AP does the final review, he is the expert here.


Thanks, Rick!

@aprepo Could you please review these changes to our docs? Thanks!

File: docs/platform/concepts/tax-information.rst [hunk 1]
```diff
@@ -8,17 +8,11 @@ None of our marketed prices include value-added or other taxes.
...
 
-For business customers in EU countries, other than Finland, we can apply the reverse charge mechanism of 2006/112/EC article 196, 
-when a valid VAT ID is entered into the billing information of the relevant Aiven project.
-
-Rest of the world
------------------
-
-No value-added tax is applied to services sold outside the European Union as described in Value-Added Tax Act of Finland section 69h. 
-Finland has tax treaties with most countries in the world, affirming this status.
+For business customers in EU countries other than Finland we can apply the reverse charge mechanism of 2006/112/EC article 196
```
Comment (wojcik-dorota) [inline docs/platform/concepts/tax-information.rst:14]: minor thing - how about a comma after "Finland" for better readability?

File: docs/platform/concepts/tax-information.rst [hunk 2]
```diff
@@ -27,21 +21,8 @@ According to the tax treaty between Finland and United States no tax needs to be
...
-.. important::
-
-    Please be aware that you can set up Billing profile per project or create a Billing Group if you want to group costs and consolidate invoices by different business units.
-    For further information please refer to :doc:`Billing Groups </docs/platform/concepts/billing-groups>`. 
+Rest of the world
+------------------
 
-Once the billing country has been updated in the billing section, it will be used in all future invoices. 
-Billing estimates are updated approximately once an hour, so it may take up to an hour for the new tax status to become visible on the invoice estimates.
+No value-added tax is applied to services sold outside the European Union as described in Value-Added Tax Act of Finland section 69h.
```
Comment (wojcik-dorota) [inline docs/platform/concepts/tax-information.rst:27]: Can we get a link to Value-Added Tax Act of Finland section 69h? Also, can we use "EU" instead a full form?

File: docs/platform/howto/use-billing-groups.rst [hunk 3]
```diff
@@ -1,63 +1,29 @@
...
 Manage billing groups
----------------------
-
-To view and manage billing groups in the `Aiven Console <https://console.aiven.io/>`_:
-
-#. Click the current project and select **See all projects & accounts**.
+=====================
 
-#. Click an account that you want to manage and then click the **Billing** tab.
+View and manage billing groups in the Aiven Console
```
Comment (wojcik-dorota) [inline docs/platform/howto/use-billing-groups.rst:4]: I'd reword the doc title into "Manage billing groups in the Aiven Console".
Also, I'd reword the first section title into "View billing groups" since there's noting about managing and it's already stated that's done in the console (see the new doc title).

File: docs/platform/howto/use-billing-groups.rst [hunk 4]
```diff
@@ -1,63 +1,29 @@
...
 #. Click one of the listed billing groups to browse the details for that billing group.
 
-Navigate the billing group page
-'''''''''''''''''''''''''''''''
-
-The billing group page contains multiple sections:
-
-* the **accumulated costs** of all projects that are assigned to the billing group in the top-right corner.
-* the *action menu* , available in the top-right corner, includes options to **rename or delete the billing group**. 
+Billing group page overview
```
Comment (wojcik-dorota) [inline docs/platform/howto/use-billing-groups.rst:11]: I'd replace this section (which most likely could be added as a concept judging by the title) with a few how-to sections covering, e.g.:
- Rename a billing group
- Delete a billing group
- And so on for all the relevant tabs.
These can be really short and simplified - what I mean is that the user expect to get some instructions here.
Ofc I'd also sneak in some general info on the page using notes, for example.

File: docs/platform/howto/update-tax-status.rst [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+You can add your VAT ID in the billing information in the Aiven Console:
+
+1. Click **Billing** and select the billing group that you want to update.
+2. Click **Billing information**.
+3. In the **Billing Details** section, click **Edit**.
+4. Enter your VAT ID and click **Save**. 
+
+.. important::
+
+    Please be aware that you can set up one billing profile per project or create a Billing Group to group costs and consolidate invoices by different business units.
```
Comment (wojcik-dorota) [inline docs/platform/howto/update-tax-status.rst:13]: I wouldn't use "please" in tech docs.

File: docs/platform/howto/update-tax-status.rst [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    For further information please refer to :doc:`Billing Groups </docs/platform/concepts/billing-groups>`.
```
Comment (wojcik-dorota) [inline docs/platform/howto/update-tax-status.rst:14]: You can use the `seealso` directive for this one. In any case, I'd remove "please" and use a comma before "refer".

File: docs/platform/howto/update-tax-status.rst [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+2. Click **Billing information**.
+3. In the **Billing Details** section, click **Edit**.
+4. Enter your VAT ID and click **Save**. 
+
+.. important::
+
+    Please be aware that you can set up one billing profile per project or create a Billing Group to group costs and consolidate invoices by different business units.
+    For further information please refer to :doc:`Billing Groups </docs/platform/concepts/billing-groups>`. 
+
+If needed, you can change the billing address by clicking **Edit** in the **Company  Details** section. Once the billing country has been updated, it will be used in all future invoices.
```
Comment (wojcik-dorota) [inline docs/platform/howto/update-tax-status.rst:16]: I'd get rid of the future tense here.

File: docs/platform/howto/update-tax-status.rst [hunk 8]
```diff
@@ -0,0 +1,17 @@
+Update your tax status
+======================
+
```
Comment (wojcik-dorota) [inline docs/platform/howto/update-tax-status.rst:3]: Are there any prerequisites for this operation?

File: docs/platform/howto/update-tax-status.rst [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+You can add your VAT ID in the billing information in the Aiven Console:
+
+1. Click **Billing** and select the billing group that you want to update.
```
Comment (wojcik-dorota) [inline docs/platform/howto/update-tax-status.rst:6]: How about using "select" instead of "click"?

File: docs/platform/howto/update-tax-status.rst [hunk 10]
```diff
@@ -0,0 +1,17 @@
+Update your tax status
```
Comment (wojcik-dorota) [inline docs/platform/howto/update-tax-status.rst:1]: I'd be good to start this article with a short statement on what the article covers and what the user can learn from this doc.

File: docs/platform/howto/create-billing-groups.rst [hunk 11]
```diff
@@ -0,0 +1,36 @@
+Create billing groups
+=====================
+
+With billing groups you can set up billing profiles to be used across multiple projects. A consolidated invoice is created for each billing group. To learn more about the key benefits of billing groups read the :doc:`related documentation <../concepts/billing-groups>`.
```
Comment (wojcik-dorota) [inline docs/platform/howto/create-billing-groups.rst:4]: I'd be good to add somewhere here what the user can learn from this article, although it may seems obvious.

File: docs/platform/howto/create-billing-groups.rst [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+---------------------------------
+
+An Aiven **Account** is the entry point for creating and managing billing groups:
+
+- Within an account you can create one or more billing groups.
+- You can assign an account's projects to a billing group and move projects from one billing group to another.
+
+Aiven credits are available for all projects associated with the account. You can choose to apply the credits to a specific project within the account.
+
+Create a new billing group
```
Comment (wojcik-dorota) [inline docs/platform/howto/create-billing-groups.rst:17]: I'd skip "new".

File: docs/platform/howto/create-billing-groups.rst (hunk 11 above)
Comment (wojcik-dorota) [inline docs/platform/howto/create-billing-groups.rst:4]: Regarding: To learn more about the key benefits of billing groups read the :doc:`related documentation <../concepts/billing-groups>`.

- I'd use a comma after "groups".
- I'd use the `seealso` directive.

File: docs/platform/howto/create-billing-groups.rst [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+--------------------------
+
```
Comment (wojcik-dorota) [inline docs/platform/howto/create-billing-groups.rst:19]: Any prerequisites for this operation?

File: docs/platform/howto/create-billing-groups.rst [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+#. In the account that you want to add a billing group to, click **Billing**.
+
+#. Click **Create billing group**.
```
Comment (wojcik-dorota) [inline docs/platform/howto/create-billing-groups.rst:22]: How about "select" instead of "click"? (this comment applies to the whole PR)

File: docs/platform/howto/create-billing-groups.rst [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+#. Enter a name for the billing group and click **Continue**.
+
+#. Enter the billing details. You can also copy these details from another billing group by selecting it from the list. Click **Continue**.
```
Comment (wojcik-dorota) [inline docs/platform/howto/create-billing-groups.rst:26]: I'd exclude the 2nd sentence from the step and include it as a note.

File: docs/platform/howto/create-billing-groups.rst [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+
+#. Select the projects that you want to add to this billing group. You can also skip this and add projects later. Click **Continue**.
+
+#. Check the information on the **Summary**. You can edit the name, information, or projects by clicking **Edit**.
```
Comment (wojcik-dorota) [inline docs/platform/howto/create-billing-groups.rst:30]: I'd remove "the" and rephrase it into: Check the information on **Summary** and **Edit** to change the name, information, or projects if needed.

File: docs/platform/howto/create-billing-groups.rst [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+
+#. When you have confirmed everything is correct, click **Create**.
+
+The costs of the projects assigned to this billing group will be consolidated into a single :doc:`invoice</docs/platform/howto/use-billing-groups>`.
```
Comment (wojcik-dorota) [inline docs/platform/howto/create-billing-groups.rst:34]: I'd refrain from using future tenses in tech docs whenever we want to generalize over something (this comment applies to the whole PR).

File: docs/platform/howto/create-billing-groups.rst (hunk 14 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/create-billing-groups.rst:22]: I won't update each of these here since "click" is used extensively throughout the docs and I'd like us to set a firm guideline before changing something (so I don't have to change it back later :)). But, I agree this is something we should decide and include as part of the style guide!
